extensions/jdbc/jdbc-derby/runtime/pom.xml

```diff
@@ -26,11 +26,6 @@
             <groupId>org.apache.derby</groupId>
             <artifactId>derbyclient</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.graalvm.sdk</groupId>
-            <artifactId>graal-sdk</artifactId>
-            <scope>provided</scope>
-        </dependency>
```

Comment:
I suppose you remove it when there's no substitutions around?

Reply:
Correct